.github/workflows/pull_request.yml

```diff
@@ -85,7 +108,7 @@ jobs:
     name: 🐪 Check Perl
     needs: build_backend
     runs-on: ubuntu-latest
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && needs.filter.outputs.code_changed == 'true'
```

Comment:
No, you should not do this because this check must pass for PR to be merged.

Currently
![image](https://github.com/user-attachments/assets/8bdca42f-f539-4ce4-80bc-6dcd3945feec)

I imagine we could skip the steps instead ?

Reply:
Or am I wrong ?
from https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks
> A job that is skipped will report its status as "Success". It will not prevent a pull request from merging, even if it is a required check.

Reply:
Indeed, I see now this was the case for this PR !

(maybe some times ago it was not the case, because I remember being block by that in the past…)